group.py

```diff
@@ -2,4 +2,29 @@
 
 # Your code to go here...
 
-my_group =
+class Person:
+
+    def __init__(self, name, age,occupation):
+        self.name = name
+        self.age = age
+        self.occupation = occupation
+        self.relationships = {}
+
+    def add_relationship(self, connectedname, relationship):
+        self.relationships[connectedname]= relationship 
+        
+        
+    
+Jill = Person("Jill", 26, "biologist")
```

Comment:
Using a class rather than a dict (as in other solutions) makes these definitions of each person really clear and easy-to-read.